DataFormats/Headers/test/dataHeaderTest.cxx

```diff
@@ -7,51 +7,6 @@
 namespace AliceO2 {
   namespace Header {
 
-    BOOST_AUTO_TEST_CASE(Description_test)
```

Comment:
that is surely a mistake, removes all changes made in 1f1700b7fdbd2a30ede283ddd9a9cc9693d3e5f9. This is definitely an incorrectly resolved merge conflict.

As it looks now, commit f40eb44aa31b5e97b373044d4bdc9b49a6800a54 must be removed. Actually Sandro's change was added in #241.